torch_geometric/nn/models/gnn_explainer.py

```diff
@@ -95,71 +85,34 @@ def __set_masks__(self, x, edge_index, init="normal"):
             self.node_feat_mask = torch.nn.Parameter(torch.randn(1, F) * std)
 
         std = torch.nn.init.calculate_gain('relu') * sqrt(2.0 / (2 * N))
-        self.edge_mask = torch.nn.Parameter(torch.randn(E) * std)
-        if not self.allow_edge_mask:
-            self.edge_mask.requires_grad_(False)
-            self.edge_mask.fill_(float('inf'))  # `sigmoid()` returns `1`.
 
-        set_masks(self.model, self.edge_mask, edge_index, apply_sigmoid=True)
+        if self.allow_edge_mask:
+            self.edge_mask = torch.nn.Parameter(torch.randn(E) * std)
 
-    def __clear_masks__(self):
+    def _clear_masks(self):
         module = clear_masks(self.model)
         self.node_feat_masks = None
         self.edge_mask = None
         module.loop_mask = None
```

Comment:
I could not find `module.loop_mask` anywhere else in the code. So can we remove this line?

Reply:
Yes, I guess it refers to `MessagePassing._loop_mask`.